client/components/category-pill-navigation/index.tsx

```diff
@@ -23,12 +24,14 @@ type CategoryPillNavigationProps = {
 		link: string;
 	}[];
 	selectedCategoryId: string;
+	onSelect?: ( selectedId: string ) => null;
 };
 
 export const CategoryPillNavigation = ( {
 	buttons,
 	categories,
 	selectedCategoryId,
+	onSelect,
```

Comment:
I suppose you should add fallback as empty function, otherwise we will have error inside `onClick` handler, if it doesn't exist